### PR TITLE
fix figsize issue; allow for str in addition to Path

### PR DESCRIPTION
**Pull request by cod3licious**
this should fix issue #1622. additionally in a bunch of places it is now possible to give file paths as strings instead of only Path instances. besides that some minor formatting things where flake8 in my text editor was being annoying.

### CODE REVIEW

File: flair/file_utils.py [hunk 1]
```diff
@@ -99,15 +101,17 @@ def cached_path(url_or_filename: str, cache_dir: Path) -> Path:
         )
 
 
-def unzip_file(file: Path, unzip_to: Path):
+def unzip_file(file: Union[str, Path], unzip_to: Union[str, Path]):
     from zipfile import ZipFile
 
-    with ZipFile(file, "r") as zipObj:
+    with ZipFile(Path(file), "r") as zipObj:
```
Comment (alanakbik) [inline flair/file_utils.py:107]: What if `file is` already a Path?

Comment (cod3licious) [reply flair/file_utils.py:107]: `Path(Path("lkasjdf/lkasjdf"))` executes without error so it shouldn't matter. Indeed probably all the checks for `str` types before conversion could probably be removed but I checked that too late 😅

Comment (alanakbik) [reply flair/file_utils.py:107]: oha didn't know that :D

**Comment by alanakbik** [comment]
:+1:

**Comment by alanakbik** [comment]
@cod3licious thanks for fixing this!

**Pull request merged**